dataactcore/utils/jsonResponse.py

```diff
@@ -41,10 +47,10 @@ def error(exception, errorCode, **kwargs):
             responseDict[key] = kwargs[key]
 
 
-        exc_type, exc_obj, exc_tb = sys.exc_info()
+        _, _, exc_tb = sys.exc_info()
```

Comment:
Not sure if this is better or not, but I think an alternate way to extract the traceback from an exception in Python 3 is `exception.__traceback__`